platform/ios/scripts/publish.sh

```diff
@@ -35,4 +35,15 @@ zip -r ../${ZIP} *
 step "Uploading ${ZIP} to s3…"
 REPO_NAME=$(basename $TRAVIS_REPO_SLUG)
 aws s3 cp ../${ZIP} s3://mapbox/$REPO_NAME/ios/builds/ --acl public-read
-echo http://mapbox.s3.amazonaws.com/$REPO_NAME/ios/builds/${ZIP}
+echo "URL: https://mapbox.s3.amazonaws.com/$REPO_NAME/ios/builds/${ZIP}"
+
+#
+# update nightly
+#
+if [[ ${PUBLISH_VERSION} =~ "nightly" ]]; then
+    step "Updating ${PUBLISH_VERSION} to ${PUBLISH_STYLE}…"
+    GENERIC_NIGHTLY_FILENAME="mapbox-ios-sdk-${PUBLISH_VERSION}.zip"
+    aws s3 cp \
+        s3://mapbox/$REPO_NAME/ios/builds/${ZIP} \
+        s3://mapbox/$REPO_NAME/ios/builds/${GENERIC_NIGHTLY_FILENAME} --acl public-read
```

Comment:
Hrm, HTTP 200 appears to be the intended behavior for _this endpoint_, [per this thread](https://forums.aws.amazon.com/thread.jspa?threadID=110336):

>The redirection functionality is only available for S3 static websites, which unfortunately means that you are seeing the intended behavior. A request served via the usual S3 endpoint will result in a response with the x-amz-website-redirect-location header, while the S3 website endpoint returns an HTTP redirect. Changing the origin to mybucketname.s3-website-us-east-1.amazonaws.com instead of using the bucket directly can theoretically solve the problem.

Hitting a [_website endpoint_](http://mapbox.s3-website-us-east-1.amazonaws.com/mapbox-gl-native/ios/builds/mapbox-ios-sdk-nightly-dynamic.zip) confirms we’re not set up as a website on s3.